### PR TITLE
Add flexbox froggy and EAK websites to learning section

### PR DESCRIPTION
**Pull request by drissboumlik**

### CODE REVIEW

File: pages-activities/activities-learning.html [hunk 1]
```diff
@@ -174,4 +174,23 @@ <h4>
             </div>
         </div>
     </div>
+    <div class="col-lg-4 col-md-6 d-flex align-items-stretch">
+        <div class="row">
+            <div class="card">
+                <h4>
+                    <span class="fas fa-frog"></span>
+                    <a href="https://flexboxfroggy.com/" target="_blank">FlexBox Froggy</a>
+                    <span class="badge badge-warning">Easy</span>
```
Comment (dangerworm) [inline pages-activities/activities-learning.html]: Easy activities are usually highlighted green, which uses the 'badge-success' class.

File: pages-activities/activities-learning.html [hunk 2]
```diff
@@ -174,4 +174,23 @@ <h4>
...
+                </h4>
+                <p>
+                    Flexbox Froggy is a game for learning CSS flexbox.<br/>
+                    It takes inspiration from the classic arcade game 
+                    <a href="https://en.wikipedia.org/wiki/Frogger">Frogger</a>, 
+                    and web literacy games like the brilliant 
+                    <a href="https://flukeout.github.io/" style="white-space: nowrap;">CSS Diner</a> 
+                    and <a href="https://eraseallkittens.com/">Erase All Kittens</a>,
```
Comment (dangerworm) [inline pages-activities/activities-learning.html:191]: We already have CSS Diner listed, but would you mind adding Erase All Kittens as well?

**Review by dangerworm — CHANGES_REQUESTED** [review]

File: pages-activities/activities-learning.html (hunk 1 above)
Comment (drissboumlik) [reply pages-activities/activities-learning.html]: Done

File: pages-activities/activities-learning.html (hunk 2 above)
Comment (drissboumlik) [reply pages-activities/activities-learning.html:191]: Done too :)

**Review by dangerworm — CHANGES_REQUESTED** [review]
Looks good. Please change target branch to `release` and I'll merge it.

**Review by dangerworm — APPROVED** [review]

**Pull request merged**